library/ortools/deploy/BUILD

```diff
@@ -0,0 +1,47 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary")
+load(":deps.bzl", "version")
+load("//distribution:deployment.bzl", "deployment")
+
+# fills the deployment url and OR-Tools version with 'sed'
+genrule(
+    name = "script",
+    srcs = ["Deploy.template.kt"],
+    outs = ["Deploy.kt"],
+    cmd = "sed '{}; {}' $(SRCS) > $(OUTS)"
+        .format(
+            "s/{google_or_tools_version}/" + version + "/g",
+            "s/{repository}/" + deployment['maven.release'].replace('/', '\/') + "/g"
+        )
+)
+
+kt_jvm_binary(
+    name = "maven",
+    main_class = "library.ortools.deploy.DeployKt",
+    srcs = [":script"],
+    deps = ["@maven//:org_zeroturnaround_zt_exec"],
+    data = [
+        "@google_or_tools_darwin//:pom-runtime.xml",
+        "@google_or_tools_darwin//:ortools-darwin-" + version + ".jar",
+        "@google_or_tools_darwin//:ortools-darwin-" + version + "-sources.jar",
+        "@google_or_tools_darwin//:pom-local.xml",
+        "@google_or_tools_darwin//:ortools-java-" + version + ".jar",
+        "@google_or_tools_darwin//:ortools-java-" + version + "-sources.jar",
+        "@google_or_tools_darwin//:ortools-java-" + version + "-javadoc.jar",
+
+        "@google_or_tools_linux//:pom-runtime.xml",
+        "@google_or_tools_linux//:ortools-linux-x86-64-" + version + ".jar",
+        "@google_or_tools_linux//:ortools-linux-x86-64-" + version + "-sources.jar",
+        "@google_or_tools_linux//:pom-local.xml",
+        "@google_or_tools_linux//:ortools-java-" + version + ".jar",
+        "@google_or_tools_linux//:ortools-java-" + version + "-sources.jar",
+        "@google_or_tools_linux//:ortools-java-" + version + "-javadoc.jar",
+
+        "@google_or_tools_windows//:pom-runtime.xml",
+        "@google_or_tools_windows//:ortools-win32-x86-64-" + version + ".jar",
+        "@google_or_tools_windows//:ortools-win32-x86-64-" + version + "-sources.jar",
+        "@google_or_tools_windows//:pom-local.xml",
+        "@google_or_tools_windows//:ortools-java-" + version + ".jar",
+        "@google_or_tools_windows//:ortools-java-" + version + "-sources.jar",
+        "@google_or_tools_windows//:ortools-java-" + version + "-javadoc.jar",
+    ]
```

Comment:
Feels like there's a fair bit of duplication here, not clear how to avoid it though

Reply:
What do you say about this approach: https://github.com/graknlabs/dependencies/pull/216/commits/08944c10fdf20b9841a698ac3f749c39b67a8b9a

Reply:
with an additional renaming here: https://github.com/graknlabs/dependencies/pull/216/commits/fed7ef86c4bfed978c1f0bc7be337805f686f2c9